scripts/deployment/deploy.sh

```diff
@@ -58,7 +58,9 @@ case $opt in
 					export COMMIT_ID=${COMMIT_ID}
 					eval $(aws ecr get-login --no-include-email)
 					aws s3 cp s3://cloudcv-secrets/evalai/${env}/docker_${env}.env ./docker/prod/docker_${env}.env
-                    			aws s3 cp s3://cloudcv-secrets/evalai/${env}/alert_manager.yml ./monitoring/prometheus/alert_manager.yml
+                    			aws s3 cp s3://cloudcv-secrets/evalai/${env}/alertmanager.yml ./monitoring/prometheus/alertmanager.yml
```

Comment:
@burnerlee why is the alert manager file name incorrect in this file? This should be `alert_manager.yml`